### PR TITLE
Render place details

### PR DESCRIPTION
**Pull request by Karyum**
#126

### CODE REVIEW

File: views/add_event.hbs [hunk 1]
```diff
@@ -236,4 +237,5 @@
     <button type="submit" name="submitBtn" class="submit-event-button">Submit</button>
   </form>
 
+<script src="https://unpkg.com/axios/dist/axios.min.js"></script>
```
Comment (Karyum) [inline views/add_event.hbs]: i'll get rid of this script when i make a PR for issue #129

File: views/event_details.hbs [hunk 2]
```diff
@@ -19,10 +19,25 @@
...
 
-  <div class="margins">
-    <h3 class="dark-text">
+  <div>
+    <h1 class="dark-text" style="font-size: 1.8rem; display: inline-block; margin-right: 1rem;">Place: {{placeDetails.name}}</h1>
+    {{#if website}}
+      {{#if facebook}}
+          <a href={{placeContact.website}}><i style="margin-bottom: 1rem; color: #409a93;" class="fa fa-facebook-square fa-2x"></i></a>
+      {{else}}
+        <a href={{placeContact.website}}><i style="margin-bottom: 1rem; color: #409a93;" class="fa fa-link fa-2x"></i></a>
```
Comment (finnhodgkin) [inline views/event_details.hbs]: Add text to indicate it's a link to the website and to make it more clickable

**Comment by Karyum** [comment]
@finnhodgkin So sorry not sure what happened :disappointed_relieved:

File: views/event_details.hbs (hunk 2 above)
Comment (Karyum) [reply views/event_details.hbs]: I feel like the icon should be self explanatory no need to add text to make it clearer.

Comment (finnhodgkin) [reply views/event_details.hbs]: I disagree and I'm pretty sure @matthewdking did too :upside_down_face: For Facebook the logo is definitely enough, but I think for a website it should at least say something like "visit our website" or just "website" along with the link icon.

Comment (finnhodgkin) [reply views/event_details.hbs]: Merge conflicts and also this.

Comment (Karyum) [reply views/event_details.hbs]: don't you think that's a bit weird though ?? to have a label next to the icon ?

website: :link:

Comment (finnhodgkin) [reply views/event_details.hbs]: I was thinking the other way round: [:link: website](http://google.com)

Comment (finnhodgkin) [reply views/event_details.hbs]: It's not so universal a symbol for website. I'm not sure a random user would know what it meant instantly. Maybe if the link icon was just to the right of the place name and the name itself was also a link, but as is I'm not sure it's clear enough.

Comment (Karyum) [reply views/event_details.hbs]: ahh okay this looks good now 
so what we gonna have is --->    Place: Haifa :link: [website](suckmyass)

Comment (finnhodgkin) [reply views/event_details.hbs]: Or even **event name** :link:

Comment (Karyum) [reply views/event_details.hbs]: ok wow now we have 3 options choose one :laughing: 

1) Place: Haifa :link: [website](suckmyasss)

2) **event name** :link: 

3) not sure how it was in the start

**Comment by Karyum** [comment]
@finnhodgkin right now it's looks like the first option from [this comment](https://github.com/foundersandcoders/nazareth-events/pull/130#discussion_r153014753) probably gonna go to Liwan and ask them whats the best thing to do.

also i just pushed a helper for when the user adds a website without the http/https protocol

File: views/event_details.hbs [hunk 3]
```diff
@@ -19,10 +19,26 @@
...
-  <div class="margins">
-    <h3 class="dark-text">
+  <div>
+    <h1 class="dark-text" style="font-size: 1.8rem; display: inline-block; margin-right: .5rem;">Place: {{placeDetails.name}}</h1>
+    {{#if website}}
+      {{#if facebook}}
+          <a href="http://www.{{removeProtocol website}}"><i style="margin-bottom: 1rem; color: #409a93;" class="fa fa-facebook-square fa-2x"></i></a>
+      {{else}}
+        <a href="http://www.{{removeProtocol website}}"><i style="margin-bottom: 1rem; color: #409a93;" class="fa fa-link fa-2x"></i></a>
+        <a href="http://www.{{removeProtocol website}}" style="font-size: 1.7rem;">website</a>
```
Comment (finnhodgkin) [inline views/event_details.hbs]: I'd prefer one wrapping `a` tag with a span for 'website':

```html
<a href="http://www.{{removeProtocol website}}">
  <i style="margin-bottom: 1rem; color: #409a93;" class="fa fa-link fa-2x"></i>
  <span style="font-size: 1.7rem;">website</span>
</a>
```

**Comment by Karyum** [comment]
@finnhodgkin :shipit: :taco: :burrito:

**Pull request merged**